frontend/simple/views/SignUp.vue

```diff
@@ -65,35 +69,64 @@
 </style>
 
 <script>
-var serialize = require('form-serialize')
-var request = require('superagent')
+import Vue from 'vue'
+import * as Events from '../../../shared/events'
+import * as db from '../js/database'
+import multihash from 'multihashes'
+import {HapiNamespace} from '../js/backend/hapi'
+var namespace = new HapiNamespace()
 
 // TODO: fix all this
 export default {
   name: 'SignUp',
   methods: {
-    submit: function () {
-      console.log(this.errors)
-      this.response = ''
-      request.post(`${process.env.API_URL}/user/`)
-      .send(serialize(this.$refs.form, {hash: true}))
-      .end((err, res) => {
-        this.error = !!err || !res.ok
-        console.log('this.error', this.error)
-        this.response = this.error ? res.body.message : (res.text === '' ? 'success' : res.text)
-        if (!this.error && this.$route.query.next) {
-          this.login()
-          this.$router.push({path: this.$route.query.next})
+    submit: async function () {
+      try {
+        let user = new Events.IdentityContract({
+          attributes: [
+            {name: 'name', value: this.name},
+            {name: 'email', value: this.email}
+          ]
+        })
+        await namespace.register(this.name, user.toHash())
+        await db.storeLogin(user.toHash(), multihash.toB58String(multihash.encode(new Buffer(this.password), 'blake2b')))
```

Comment:
OK, for now can you just directly store the password + add a TODO comment? That way I can hook in the crypto stuff later when it's ready, as simply storing the hash isn't secure (and this is also not how one would do it, i.e. it's missing the actual black2 stuff).